php/admin/class-settings.php

```diff
@@ -101,9 +104,12 @@ public function render_page_header() {
 		?>
 		<h2><?php echo esc_html( get_admin_page_title() ); ?></h2>
 		<h2 class="nav-tab-wrapper">
-			<a href="<?php echo esc_url( add_query_arg( 'tab', 'license' ) ); ?>" title="<?php esc_attr_e( 'License', 'block-lab' ); ?>" class="nav-tab nav-tab-active dashicons-before dashicons-nametag">
+			<a href="<?php echo esc_url( add_query_arg( 'tab', 'license' ) ); ?>" title="<?php esc_attr_e( 'License', 'block-lab' ); ?>" class="nav-tab <?php echo ( ! isset( $_GET['tab'] ) || 'license' === $_GET['tab'] ) ? 'nav-tab-active' : ''; //phpcs:ignore ?> dashicons-before dashicons-nametag">
```

Comment:
You can use `filter_input` here. See how we do it:
https://github.com/getblocklab/block-lab/blob/d9f5a827cd51652766c343ed2768f0490fd79cb9/php/admin/class-settings.php#L42

Reply:
Removed settings screen, so this no longer applies.